default/python/AI/freeorion_tools/chat_handler/interpreter.py

```diff
@@ -18,6 +18,8 @@ def eval(self, msg):
 
         sys.stdout = StringIO()
         sys.stderr = StringIO()
+        if msg.endswith(';'):
+            msg = msg.replace(';', '\n')
```

Comment:
This seems potentially risky, but since it's effectively opt-in, I suppose it's fine. If @Morlic-fo or @Cjkjvfnby object it can be reverted...

Reply:
I'm reluctant to merge if the black linter is throwing an error, even though there's no indication what the problem is...

Reply:
Just installed black. That tool isn't very talkative:
black AI/freeorion_tools/chat_handler/interpreter.py --verbose
Using configuration from /home/muzz/freeorion-project/Grummel7/pyproject.toml.
reformatted AI/freeorion_tools/chat_handler/interpreter.py
All done! ✨ 🍰 ✨
1 file reformatted.

It didn't like the single-quotes. I will update the readme. It already says that you should use flake8, but black it not mentioned.